content/en/docs/Writing policies/generate.md

```diff
@@ -26,6 +26,19 @@ Kubernetes has many default resource types even before considering CustomResourc
 When generating a custom resource, it is necessary to set the apiVersion (ex., `spec.generate.apiVersion` and kind (ex., `spec.generate.kind`).
 {{% /alert %}}
 
+Kyverno will create an intermediate object called a `GenerateRequest` which is used to queue work items for the final resource generation. To get the details and status of a generated resource, check the details of the generate request. The following will give the list of generate requests.
+
+```sh
+kubectl get generaterequests -A
+```
+
+A generate request status can have one of four values:
+
+`Completed`: the generate request controller created resources defined in the policy
+`Failed`: the generate request controller failed to process the rules
+`Pending`: the request is yet to be processed or the resource has not been created
+`Skip`: the generate request controller has skipped the creation of resource
```

Comment:
As of now, the `Skip` is marked when triggering the generate policy by adding the label/annotation to the existing resource, while the selector is not defined in the policy itself.

It can be extend to cover other scenarios, cc @prateekpandey14.